services/worker-manager/src/providers/aws.js

```diff
@@ -163,6 +162,7 @@ class AwsProvider extends Provider {
           state: this.Worker.states.REQUESTED,
           providerData: {
             region: config.region,
+            instanceCapacity: config.capacityPerInstance,
```

Comment:
:heart: this is what always tripped up aws-provisioner: it didn't know the capacity of existing instances.  Nice to see that solved here!